### DISCUSSION TITLE
One Artist, Multiple Albums

### DISCUSSION THREAD

**Discussion by jyrg02** [beetbox/beets#4602] [Q&A] [answered]
Usually I have this case where I have one single artist and various songs from different albums. Therefore I have two questions What Is there any way to import them so that I get the album cover? Since the plugin of album covers only works with tracks viewed as a whole. Also when doing it this way it takes all the tracks in the folder and attempts to do its best to match them...regardless of whether they are from the same artist or not. Thus I in the end have mislabelled songs and artists 
### This is the code that is spit out 
`C:\Users\Lenovo\Music\newmusic (8 items)
Finding tags for album "Sabaton - ".
Candidates:
1. BLACKPINK - THE ALBUM (27.2%) (tracks, album, artist, ...) (Spotify, 2020, YG Entertainment/Interscope Records)
2. ABBA - The Album (27.1%) (tracks, album, artist, ...) (Spotify, 1977, Polar Music International AB)
3. Peter, Paul and Mary - Album 1700 (25.8%) (tracks, missing tracks, album, ...) (Spotify, 1967, Warner Records)
4. San Holo - album1 (25.3%) (tracks, missing tracks, album, ...) (Spotify, 2018, bitbird)
5. The Jackson 5 - Christmas Album (23.8%) (tracks, album, missing tracks, ...) (Spotify, 1970, UNI/MOTOWN)
6. Kendrick Lamar, SZA - Black Panther The Album Music From And Inspired By (23.3%) (tracks, missing tracks, album, ...) (Spotify, 2018, Black Panther (TDE/DMG) PS)
7. Dean Martin - The Dean Martin Christmas Album (22.8%) (tracks, album, missing tracks, ...) (Spotify, 1966, Legacy Recordings)
8. Los Acosta - Album de Oro (21.9%) (tracks, missing tracks, album, ...) (Spotify, 1995, MCM Mexico/WMI)
9. Andy Williams - The Andy Williams Christmas Album (21.8%) (tracks, missing tracks, album, ...) (Spotify, 1963, Columbia/Legacy)
10. Elvis Presley - Elvis' Christmas Album (21.7%) (tracks, missing tracks, album, ...) (Spotify, 1957, RCA Victor)
11. Teyana Taylor - The Album (21.5%) (missing tracks, tracks, album, ...) (Spotify, 2020, Getting Out Our Dreams, Inc./Def Jam Recordings)
12. The Beach Boys - The Beach Boys' Christmas Album (21.4%) (tracks, missing tracks, album, ...) (Spotify, 1964, Capitol Records)
13. Chase Rice - The Album (21.0%) (tracks, missing tracks, album, ...) (Spotify, 2021, Broken Bow Records)
14. Jeremih - Late Nights: The Album (20.9%) (tracks, missing tracks, album, ...) (Spotify, 2015, Def Jam)
15. Cody Johnson - Human The Double Album (20.8%) (tracks, missing tracks, album, ...) (Spotify, 2021, CoJo Music/Warner Music Nashville)
16. deadmau5 - > album title goes here < (19.1%) (tracks, missing tracks, album, ...) (Spotify, 2012, Ultra Records)
17. Feid - FELIZ CUMPLEAÑOS FERXXO TE PIRATEAMOS EL ÁLBUM (18.5%) (tracks, missing tracks, album, ...) (Spotify, 2022, UMLE - Latino)
18. Morgan Wallen - Dangerous: The Double Album (16.8%) (missing tracks, tracks, album, ...) (Spotify, 2021, Big Loud / Republic)
19. Morgan Wallen - Dangerous: The Double Album (Bonus) (15.8%) (missing tracks, tracks, album, ...) (Spotify, 2021, Big Loud / Republic)
20. J. Cole - Album of the Year (Freestyle) (12.6%) (unmatched tracks, album, artist, ...) (Spotify, 2018, J. Cole P&D)
# selection (default 1), Skip, Use as-is, as Tracks, Group albums,
Enter search, enter Id, aBort, eDit, edit Candidates? e
Artist: Sabaton
Album: The Art of War
Finding tags for album "Sabaton - ".
Candidates:
1. Sabaton - The Art of War (Re-Armed) (30.7%) (tracks, missing tracks, album, ...) (Spotify, 2010, Nuclear Blast)
2. Sabaton - The Art of War (Re-Armed) (28.7%) (tracks, missing tracks, album) (CD, 2010, DE, Nuclear Blast, 27361 26452)
3. Sabaton - The Art of War (28.7%) (tracks, missing tracks, album) (Digital Media, 2010, AF, Nuclear Blast, Re‐Armed)
4. Sabaton - The Art of War (28.6%) (tracks, missing tracks, album) (12" Vinyl, 2014, DE, Nuclear Blast)
5. Sabaton - The Art of War (27.9%) (tracks, missing tracks, album) (CD, 2008, XE, Black Lodge Records, BLOD055CD)
6. Sabaton - The Art of War (27.9%) (tracks, missing tracks, album) (CD, 2008, SE, Black Lodge Records, BLOD055CDL, limited edition)
7. Angie Stone - The Art of Love & War (25.8%) (tracks, missing tracks, album, ...) (Spotify, 2007, Stax)
8. Various Artists - Enjoy Every Sandwich - The Songs of Warren Zevon (24.6%) (tracks, missing tracks, album, ...) (Spotify, 2004, Vanguard)
9. Killdozer - Uncompromising War on Art Under the Dictatorship of the Proletariat (23.8%) (tracks, album, missing tracks, ...) (Spotify, 1994, Touch and Go Records)
10. Various Artists - Take Warning: The Songs of Operation Ivy (23.0%) (tracks, missing tracks, album, ...) (Spotify, 1997, Oglio Entertainment)
11. Bone Thugs-N-Harmony - The Art of War: World War 1 (22.9%) (tracks, missing tracks, album, ...) (Spotify, 1997, Ruthless Records)
12. John Williams - Star Wars: Revenge of the Sith (Original Motion Picture Soundtrack) (22.7%) (tracks, missing tracks, album, ...) (Spotify, 2005, Walt Disney Records)
13. Bone Thugs-N-Harmony - The Art of War: World War 1 (22.7%) (tracks, missing tracks, album, ...) (Spotify, 1997, Ruthless Records)
14. American Head Charge - The War Of Art (22.2%) (tracks, missing tracks, album, ...) (Spotify, 2001, American Recordings IDJ)
15. Bone Thugs-N-Harmony - The Art of War: World War 2 (22.1%) (tracks, missing tracks, album, ...) (Spotify, 1997, Ruthless Records)
16. John Williams - Star Wars: Return of the Jedi (Original Motion Picture Soundtrack) (22.0%) (tracks, album, artist, ...) (Spotify, 1983, Walt Disney Records)
17. Bone Thugs-N-Harmony - The Art of War: World War 2 (21.5%) (tracks, missing tracks, album, ...) (Spotify, 1997, Ruthless Records)
18. John Williams - Star Wars: Attack of the Clones (Original Motion Picture Soundtrack) (21.3%) (tracks, missing tracks, album, ...) (Spotify, 2002, Walt Disney Records)
19. John Williams - Star Wars: The Rise of Skywalker (Original Motion Picture Soundtrack) (19.9%) (tracks, missing tracks, album, ...) (Spotify, 2019, Walt Disney Records)
20. Hip Kitty - Art of War (19.6%) (tracks, missing tracks, album, ...) (CD, 2008, US)
21. Various Artists - World of Warcraft: Wrath of the Lich King Original Soundtrack (19.3%) (missing tracks, tracks, album, ...) (Spotify, 2008, Azeroth Music)
22. Opus Akoben - Art of War (19.0%) (tracks, missing tracks, album, ...) (1997, US)
23. Various Artists - Songs Of The Civil War (18.3%) (missing tracks, tracks, album, ...) (Spotify, 1991, Columbia)
24. King Nine - The Art of War (18.1%) (unmatched tracks, album, artist, ...) (Spotify, 2016, Closed Casket Activities)
25. DJ Desue - Art of War (16.1%) (tracks, missing tracks, album, ...) (CD, 2002, DE, Rah Rah Entertainment, 016 851-2)
26. Various Artists - Divided & United: The Songs of the Civil War (16.1%) (missing tracks, tracks, album, ...) (Spotify, 2013, ATO Records)
27. Various Artists - Art of War (15.8%) (missing tracks, tracks, album) (CD, 2002, US, Tantor Media)
28. Primigenium - Art of War (15.1%) (tracks, album, artist, ...) (CD, 1997, US, Full Moon Productions, FMP011)
29. John Williams, Baltic House Orchestra - Star Wars: The Rise of Skywalker (Main Trailer Theme) (14.3%) (unmatched tracks, album, artist, ...) (Spotify, 2019, Baltic House Orchestra)
30. Various Artists - Guild Wars 2 (End of Dragons The Original Soundtrack) (9.6%) (missing tracks, tracks, album, ...) (Spotify, 2022, Various Artists)
# selection (default 1), Skip, Use as-is, as Tracks, Group albums,
Enter search, enter Id, aBort, eDit, edit Candidates?
Correcting tags from:
    Sabaton -
To:
    Sabaton - The Art of War (Re-Armed)
URL:
    https://open.spotify.com/album/0ByHC8GAef76h66JrvigzE
(Similarity: 30.7%) (tracks, missing tracks, album, source) (Spotify, 2010, Nuclear Blast)
 * 00 Devil Dogs (History Version) (#0) (3:36) ->
   Ghost Division (#2) (3:51) (title, source, length)
 * Unity - Nina Hagen (#0) ->
   Talvisota (#8) (title, source)
 * Defence of Moscow (#0) ->
   Union (Slopes of St.Benedict) (#10) (id, title, source)
 * A Lifetime of War (#1) ->
   The Price of a Mile (#11) (id, title, source, index)
 * Devil Dogs (history version) (#0) (3:36) ->
   Firestorm (#12) (3:25) (id, title, source, length)
 * Aces in Exile (#6) ->
   Swedish Pagans (#14) (id, title, source, index)
 * Father (#1) (4:35) ->
   Art of War - Pre Production Demos (#16) (4:48) (id, title, source, index, length)
 * 16 tons - Nina Hagen (#0) ->
   Swedish National Anthem - Live, at Sweden Rock Festival (#17) (title, source)
Missing tracks (9/17 - 52.9%):
 ! Sun Tzu Says          (# 1) (0:23)
 ! The Art of War        (# 3) (5:08)
 ! 40:1                  (# 4) (4:10)
 ! Unbreakable           (# 5) (5:58)
 ! The Nature of Warfare (# 6) (1:18)
 ! Cliffs of Gallipoli   (# 7) (5:51)
 ! Panzerkampf           (# 9) (5:15)
 ! A Secret              (#13) (0:37)
 ! Glorious Land         (#15) (3:19)`

**Comment by JOJ0** [comment, marked as answer]
You forgot to post your beet  import command.

Also it would help if you format your shell output into a codebox. It's pretty unreadable and messy currently....

**Comment by sunkuo** [comment]
谢谢您的来信，已收到！

**Comment by CompeyDev** [comment]
I'm very confused as to how I got mentioned here...?

![image](https://user-images.githubusercontent.com/74418041/209963671-8356ca32-40b2-4ab2-b137-0bd00b46e1d6.png)

**Reply by jatinderaujla** [reply]
Same for me

**Reply by TroyNeubauer** [reply]
Me too... Never interacted with this project before now

**Reply by marcoaam95** [reply]
Me too. WTF?

**Comment by JOJ0** [comment]
I have a feeling this is some kind of spam or malware thing we are dealing with. Very odd...

**Comment by suvratapte** [comment]
Looks like spam.
